scripts/vet.sh

```diff
@@ -69,6 +69,10 @@ not git grep "\(import \|^\s*\)\"google.golang.org/grpc/interop/grpc_testing" --
 # - Ensure all xds proto imports are renamed to *pb or *grpc.
 git grep '"github.com/envoyproxy/go-control-plane/envoy' -- '*.go' ':(exclude)*.pb.go' | not grep -v 'pb "\|grpc "'
 
+# - Ensure all context usages are done with timeout.
+git grep -e 'context.Background()' --or -e 'context.TODO()' -- "*_test.go" | grep -v "benchmark/primitives/context_test.go" | grep -v "credential
```

Comment:
1. I understand that you excluded `benchmark/primitives/context_test.go`, as we specifically want to benchmark scenarios where context.Background() is used. Would you mind adding a comment that explains this?

2. I also understand that you want to address excluded packages in subsequent PRs. Would you mind adding a TODO comment about this below L72, with a link to https://github.com/grpc/grpc-go/issues/7304?

Reply:
Added the clarification for the benchmark tests and a TODO for the rest of packages. 
Let me know if you have any suggestions on it.